doc/changelog.md

```diff
@@ -26,6 +27,20 @@
   123
   ```
 
+  There is also a callback to improve the interface for printing.
+  It is equivalent to JAX's `debug_print` function. E.g.,
+
+  ```py
+  @qml.qjit
+  def func(*args, **kwargs):
+     debug_print("{arg2}, {arg1}, arg2=args[1], args1=[0])
+  ```
+
+  ```pycon
+  >>> print(func(1, 2))
+  2 1
```

Comment:
```suggestion
  >>> func(1, 2)
  2 1
```

since 'console syntax' will always automatically print output :)

(technically, it will print the `repr()` rather than the `str()`, but same effect).

Reply:
From the testing, it looks like print uses string rather than repr : https://stackoverflow.com/a/36156152

Reply:
https://github.com/PennyLaneAI/catalyst/pull/621/commits/09f95c5aa3ce6575dce1b3374b2e11a5e043c561